documentation/docs/install-pmm/install-pmm-client/connect-database/mongodb.md

```diff
@@ -1,50 +1,56 @@
 # Connect MongoDB instance
 
-How to set up PMM to monitor a [MongoDB] or [Percona Server for MongoDB] database instance.
+Connect a MongoDB instance to PMM to monitor a [MongoDB] or [Percona Server for MongoDB] database server.
 
-## Before you start
+## Prerequisites
 
-Check that:
+Before you start, ensure you have:
 
-- [PMM Server is installed](../../install-pmm-server/index.md) and running with a known IP address or hostname accessible from the client node.
-- [PMM Client is installed](../../install-pmm-client/index.md) and the [nodes are registered with PMM Server](../../register-client-node/index.md).
-- You have superuser (root) access on the client host.
-- You have `adminUserAnyDatabase` or superuser role privilege to any database servers that you want to monitor.
-- Your MongoDB server is version 4.0 or higher.
+- [PMM Server installed](../../install-pmm-server/index.md) and running with a known IP address or hostname accessible from the Client node.
+- [PMM Client installed](../../install-pmm-client/index.md) and the [nodes are registered with PMM Server](../../register-client-node/index.md).
+- Superuser (root) access on the Client host.
```

Comment:
I'm not sure if we need superuser access on the client node.

Reply:
I think it is required because pmm-agent runs as root by default does it not?